ROMFS/px4fmu_common/init.d/4900_crazyflie

```diff
@@ -14,39 +14,52 @@
 #
 # @maintainer Dennis Shtatov <densht@gmail.com>
 #
-
 sh /etc/init.d/4001_quad_x
-
-
 if [ $AUTOCNF == yes ]
 then
-	param set COM_RC_IN_MODE 2
-	param set BAT_N_CELLS 1
-	param set BAT_CAPACITY 240
-	param set BAT_SOURCE 1
-
-	param set PWM_DISARMED 0
-	param set PWM_MIN 0
-	param set PWM_MAX 255
-
-	param set SYS_COMPANION 20
-
-	param set MC_PITCHRATE_D 0.0015
-	param set MC_PITCHRATE_I 0.05
-	param set MC_PITCHRATE_P 0.045
-	param set MC_PITCH_P 6.5
-	param set MC_ROLLRATE_D 0.0015
-	param set MC_ROLLRATE_I 0.05
-	param set MC_ROLLRATE_P 0.045
-	param set MC_ROLL_P 6.5
-	param set MC_YAW_P 3.0
-
-	param set CBRK_SUPPLY_CHK 894281
-	param set CBRK_USB_CHK 197848
+    param set COM_RC_IN_MODE 1
+    param set BAT_N_CELLS 1
+    param set BAT_CAPACITY 240
+    param set BAT_SOURCE 1
+    param set PWM_DISARMED 0
+    param set PWM_MIN 0
+    param set PWM_MAX 255
+    param set SYS_COMPANION 20
+    param set MC_PITCHRATE_D 0.002
+    param set MC_PITCHRATE_I 0.2
+    param set MC_PITCHRATE_P 0.07
+    param set MC_PITCH_P 6.5
+    param set MC_ROLLRATE_D 0.002
+    param set MC_ROLLRATE_I 0.2
+    param set MC_ROLLRATE_P 0.07
+    param set MC_ROLL_P 6.5
+    param set MC_YAW_P 3.0
+    param set EKF2_HGT_MODE 2
+    param set EKF2_AID_MASK 3
+    param set EKF2_OF_DELAY 10
+    param set MPC_THR_HOVER 0.7
+    param set MPC_MANTHR_MAX 1.0
+    param set MPC_THR_MAX 1.0
+    param set MPC_Z_P 1.5
+    param set MPC_Z_VEL_I 0.3
+    param set MPC_Z_VEL_P 0.4
+    param set MPC_HOLD_MAX_XY 0.1
```

Comment:
Minor suggestion, sorting these alphabetically and splitting roughly by group makes this a bit easier to read or change.